core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -492,12 +480,12 @@ case class ShardingContainerPoolBalancerState(
    * @return calculated invoker slot
    */
   private def getInvokerSlot(memory: ByteSize): ByteSize = {
-    val newTreshold = if (memory / _clusterSize < MemoryLimit.minMemory) {
+    val newTreshold = if (memory / _clusterSize < MemoryLimit.MIN_MEMORY) {
       logging.warn(
         this,
         s"registered controllers: ${_clusterSize}: the slots per invoker fall below the min memory of one action.")(
         TransactionId.loadbalancer)
-      MemoryLimit.minMemory
+      MemoryLimit.MIN_MEMORY
```

Comment:
I think that the condition `memory / _clusterSize < MemoryLimit.MIN_MEMORY` should lead to an `ERROR` log entry. If we forcefully set the controller's invoker shard memory size to `MemoryLimit.MIN_MEMORY`, chances are very high that we are going to overload invokers.